### ISSUE TITLE
Can we create Multiple Split window using one single app instance ?

### ISSUE THREAD

**Issue by ikhizarwork** [ivmartel/dwv-jqmobile#277]
For better understanding of the dicom data i want to have multiple canvas to view multiple dicom series at one time , i.e i want multiple split windows , so is it possible to have it using one single app instance only ? 

myapp = new dwv.App();

and i also want it to be in sycn with each other , i.e if a user tries to scroll then all the canvas must start scrolling .

is it possible @ivmartel  ?

**Comment by ivmartel** [comment]
Multiple views is available with dwv (see [demo](https://ivmartel.github.io/dwv/demo/stable/viewer.html?input=https%3A%2F%2Fraw.githubusercontent.com%2Fivmartel%2Fdwv%2Fmaster%2Ftests%2Fdata%2F%3Ffile%3Dbbmri-53323851.dcm%26file%3Dbbmri-53323707.dcm%26file%3Dbbmri-53323563.dcm%26file%3Dbbmri-53323419.dcm%26file%3Dbbmri-53323275.dcm%26file%3Dbbmri-53323131.dcm&dwvReplaceMode=void), check the layout select) but not in dwv-jqmobile (this repo).

**Comment by ikhizarwork** [comment]
Can we open Multiple datasets (multiple different series for comparison purpose ) using one single app instance @ivmartel ?

**Comment by ivmartel** [comment]
Yes! Check it in the dwv demo (link above): load your 2 datasets and you will see them overlap.

**Comment by ivmartel** [comment]
I hope I answered your question. Closing for now, feel free to comment if you need.
